sdks/python/apache_beam/runners/direct/clock.py

```diff
@@ -15,36 +15,35 @@
 # limitations under the License.
 #
 
-"""Clock implementations for real time processing and testing."""
+"""Clock implementations for real time processing and testing.
 
+For internal use only. No backwards compatibility guarantees.
+"""
 from __future__ import absolute_import
 
 import time
 
 
 class Clock(object):
-  """For internal use only; no backwards-compatibility guarantees."""
+  def current_time(self):
```

Comment:
<!--new_thread; commit:517fae9dcb749a59322ea40c7de0eb90ebf5efac; resolved:0-->
Can we rename this `time()` to be consistent with the way `time.time()` is called?